lib/puppeteer_setup.js

```diff
@@ -0,0 +1,16 @@
+const chalk = require('chalk')
+const puppeteer = require('puppeteer')
+const fs = require('fs')
+const mkdirp = require('mkdirp')
+const os = require('os')
+const path = require('path')
+
+const DIR = path.join(os.tmpdir(), 'jest_puppeteer_global_setup')
+
+module.exports = async function () {
+  console.log(chalk.green('Setup Puppeteer'))
+  const browser = await puppeteer.launch({args: ['--no-sandbox']})
```

Comment:
Again, comments here would be good. For example, what is `--no-sandbox` doing? Why is it needed?